accord-core/src/main/java/accord/topology/Topologies.java

```diff
@@ -298,10 +308,18 @@ public boolean contains(Id to)
             return false;
         }
 
+        @Override
+        public int estimateUniqueNodes()
```

Comment:
Since `nodes()` calculates this precisely should we remember that value and then keep returning it?

Reply:
I don't follow

Reply:
I am overthinking it. I think we always invoke `nodes` to do something useful with the `Topologies` and that calculates the precise number of unique nodes and we could then start returning it.

But's so rarely used doesn't seem worth the complexity.

Reply:
You’re right, though I think I’d be inclined to just precompute nodes and stash it somewhere instead in that case. I’d be ok with that.